### PR TITLE
UZH tokens and scripts

### PR DESCRIPTION
**Pull request by dydent**
update token contracts and deploy to uzh
update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
+the [UZH Ethereum network](http://uzheth.business.uzh.ch)
+
+## Setup
+
+In the root folder create a `.env` file with the following values:
+
+    UZH_PRIVATE_KEY="<private key of a valid UZHETH account to deploy contracts>"  
+    GANACHE_PRIVATE_KEY="<private key of a valid GANACHE account to deploy contracts>"  
+    GANACHE_URL="http://127.0.0.1:7545"
+    UZH_URL="http://130.60.244.246:8545"
```
Comment (j0nezz) [inline README.md]: http://uzheth.business.uzh.ch

File: scripts/deploy-erc20-tokens.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+async function main() {
+  let addresses = {};
+
+  // deploy UZHETH tokens
+  const UzhUniToken = await ethers.getContractFactory("UzhUniToken");
+  const uzhuni = await UzhUniToken.deploy();
+  await uzhuni.deployed();
+  console.log("Deployed UzhUniToken (UZHUNI): ", uzhuni.address);
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  addresses = { ...addresses, uzhuni: { address: uzhuni.address } };
```
Comment (j0nezz) [inline scripts/deploy-erc20-tokens.ts]: It's not React :D You can just use addresses['uzhuni']=... , no need to always spread and create a new object.

**Review by j0nezz — CHANGES_REQUESTED** [review]
JSON output is nice! Can you also add it to the Uniswap contracts?
Please also add  the addresses of the deployed ERC20 to the readme.
Thaanks 😄

File: README.md (hunk 1 above)
Comment (dydent) [reply README.md]: this link was not working when I tried it on my side :D
should I change it anyway?

Comment (j0nezz) [reply README.md]: Probably not  then 👀

**Review by j0nezz — APPROVED** [review]
Nice, only minor feedback. Network URLs don't really need to be in the .env file, because they're not sensitive.

**Pull request merged**